File: POOEncapsulamento/app/src/main/java/com/example/pooencapsulamento/Livro.java
package com.example.pooencapsulamento;

/**
 *  Classe Livro - ENcapsulamento
 *
 *  Atributos
 *
 *  - Titulo - caracteres / String
 *  - Autor - caracteres / String
 *  - Editora - caracteres / String
 *  - Paginas - numeros inteiros / int / Integer
 *  - Localizacao - caracteres / String
 *
 * Construtors - cria a instancia dos objetos da classe
 *
 * Métodos
 *
 * gets
 * sets
 *
 */

public class Livro {

    private String Titulo;
    private String Autor;
    private String Editora;
    private int Paginas;
    private String Localizacao;


    public String getTitulo() {
        return Titulo;
    }

    public void setTitulo(String titulo) {
        Titulo = titulo;
    }

    public String getAutor() {
        return Autor;
    }

    public void setAutor(String autor) {
        Autor = autor;
    }

    public String getEditora() {
        return Editora;
    }

    public void setEditora(String editora) {
        Editora = editora;
    }

    public int getPaginas() {
        return Paginas;
    }

    public void setPaginas(int paginas) {
        Paginas = paginas;
    }

    public String getLocalizacao() {
        return Localizacao;
    }

    public void setLocalizacao(String localizacao) {
        Localizacao = localizacao;
    }
}

File: POOKotlinEncapsulamento/app/src/main/java/com/example/pookotlinencapsulamento/Livro.kt
package com.example.pookotlinencapsulamento

class Livro(
    titulo: String,
    autor: String,
    editora: String,
    localizacao: String,
    paginas: Int
) {
    private var _titulo = titulo
    private var _autor = autor
    private var _editora = editora
    private var _localizacao = localizacao
    private var _paginas = paginas

    var titulo: String
        get() = _titulo
        set(valor) {
            _titulo = valor
        }

    var autor: String
        get() = _autor
        set(valor) {
            _autor = valor
        }

    var editora: String
        get() = _editora
        set(valor) {
            _editora = valor
        }

    var localizacao: String
        get() = _localizacao
        set(valor) {
            _localizacao = valor
        }

    var paginas: Int
        get() = _paginas
        set(valor) {
            _paginas = valor
        }
}